### ISSUE TITLE
cannot access apache logs

### ISSUE THREAD

**Issue by phil123456** [NarrativeScience/log.io#191]
Hello,

I install it that way

```
sudo npm install -g log.io --user "myadminuser" --unsafe-perm
sudo log.io-server &
sudo nano /root/.log.io/harvester.conf
sudo log.io-harvester &
```

but as I run harvester :

```
log.io-harvester &
[3] 3510
$ error: File doesn't exist: '/var/log/apache2/access.log'
error: File doesn't exist: '/var/log/apache2/error.log'
fs.js:1445
    throw error;
    ^

Error: watch /var/log/syslog EACCES
    at exports._errnoException (util.js:1018:11)
    at FSWatcher.start (fs.js:1443:19)
    at Object.fs.watch (fs.js:1470:11)
    at LogStream._watchFile ....
```

but the file is there, so it got to be a permission issue

```
sudo ls /var/log/apache2/access.log
/var/log/apache2/access.log
```

**Comment by phil123456** [comment]
anyone ?

**Comment by phil123456** [comment]
please anyone ? this bug makes the application unusable on debian

**Comment by msmathers** [comment]
It's likely that the `log.io-harvester` process is being run by a user that doesn't have read permissions on `/var/log/apache2/access.log`.  Do `ls -al /var/log/apache2/` to see the permissions.

Also, the newly released v0.4.3 uses a different installation/bin method for running the harvester & server, follow the new installation instructions.
